.github/workflows/ocwm-pre-meeting-check.yml

```
@@ -0,0 +1,83 @@
+name: OCWM Pre-Meeting Check
+
+on:
+  schedule:
+    - cron: '50 21 * * 1'  # Runs at 21:50 UTC, which is 10 minutes before 14:00 PT every Monday
+
+jobs:
+  check_agenda:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Set up Node 20
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+
+    - name: Get GitHub Token
+      uses: actions/create-github-app-token@v1
+      id: get_workflow_token
+      with:
+        app-id: ${{ vars.APP_ID }}
+        private-key: ${{ secrets.PRIVATE_KEY }}
+
+    - name: Install Dependencies
+      run: npm install @octokit/core@5.1.0
+
+    - name: Check Latest OCWM Issue
+      id: check_issue
+      env:
+        MY_TOKEN: ${{ steps.get_workflow_token.outputs.token }}
+        OWNER: ${{ vars.ORGANISATION }}
+        REPO: 'community'
+        OCWM_LABEL: ${{ vars.OCWM_LABEL }}
+        TEMPLATE_PATH: '../ISSUE_TEMPLATE/open_community_working_meeting.md'
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

```suggestion
        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_GEN_NOTIF }}
```

This is the secret containing the the webhook